shotover-proxy/src/transforms/redis/sink_single.rs

```diff
@@ -89,7 +98,10 @@ impl Transform for RedisSinkSingle {
 
         match outbound_framed_codec.next().fuse().await {
             Some(a) => a,
-            None => Err(anyhow!("couldnt get frame")),
+            None => {
+                counter!("redis_single_failed_request", 1, "chain" => self.chain_name.clone());
```

Comment:
This doesnt seem to match up with when the cluster sink emits this counter

Reply:
Ah I see the cluster records specifically Redis error message not just a general error.

Reply:
Aren't those "shotover generated redis errors" just actual Redis errors being passed back? 

We aren't changing the type of Redis error or anything about the message.

Reply:
ah, true!
So the functionality for the counter looks like: record every error the remote server sends back do not record any shotover side errors.